skorch/callbacks/training.py

```diff
@@ -157,6 +169,7 @@ def __init__(
         self.dirname = dirname
         self.event_name = event_name
         self.sink = sink
+        self.load_best = load_best
```

Comment:
Using `load_best=True` in conjunction with `monitor=None` wouldn't make sense. Do you think it's worth checking that combination?

Reply:
Since it does not impact the user negatively I will just add documentation and prevent the loading from occurring when `self.monitor is None` to save IO/computation.